test/npm/yarnlock_test.exs

```diff
@@ -0,0 +1,12 @@
+defmodule YarnlockTest do
+  use ExUnit.Case
+
+  test "extracts dependencies from yarn.lock" do
+    {lib_map, deps_count} = Npm.Yarnlockfile.parse!(File.read!("./test/fixtures/yarnlock"))
+
+    parsed_yarn = [{"ajv", "6.12.3"}, {"asn1", "0.2.4"}, {"assert-plus", "1.0.0"}]
```

Comment:
I didn't nit pick in the other test cases, but can we add an assertion to verify the actual library and version for one of them?

Reply:
Durr...i see it now.  Sheesh, where's my glasses?